File: problem1.js
for(var i = 1; i < 101; i++){
    if(i % 3 === 0 && i % 5 === 0){
        console.log("SenirOp");
    } else if(i % 3 === 0){
        console.log("Senir");
    } else if(i % 5 === 0){
        console.log("Op");
    } else {
        console.log(i)
    }
}

File: problem1.py
for num in range(1, 101):
    if num % 3 == 0 and num % 5 == 0:
        print('SenirOp')
    elif num % 3 == 0:
        print('Senir')
    elif num % 5 == 0:
        print('Op')
    else:
        print(num) 